chirho/robust/ops.py

```diff
@@ -127,7 +126,7 @@ def _influence_functional(*models: Callable[P, Any]) -> Callable[P, S]:
         if len(models) != len(points):
             raise ValueError("mismatch between number of models and points")
 
-        linearized = linearize(*models, **linearize_kwargs)
+        linearized = linearize(*models, **linearize_kwargs)  # type: ignore
```

Comment:
Why is this needed?

Reply:
This is unrelated / can remove this line. I added it because the linter complains that linearized isn't typed.